website/src/documentation/sdks/java-dependencies.md

```diff
@@ -26,336 +26,51 @@ behavior in the service. If you are using any of these packages in your code, be
 aware that some libraries are not forward-compatible and you may need to pin to
 the listed versions that will be in scope during execution.
 
-<p>To see the compile and runtime dependencies for your Beam SDK version, expand
-the relevant section below.</p>
+Compile and runtime dependencies for your Beam SDK version are listed in `BeamModulePlugin.groovy` in the Beam repository. To view them, perform the following steps:
 
-<details><summary markdown="span"><b>2.9.0</b></summary>
+1. Open `BeamModulePlugin.groovy`.
```

Comment:
Can we link the file to the latest file?

```
[`BeamModulePlugin.groovy`](https://raw.githubusercontent.com/apache/beam/v{{ site.release_latest }}/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy)
```

Reply:
I don't want to do it here. The point of this step is to show which file the reader should look in and which placeholder they should substitute.

The example now shows (and links) to the latest version of the file.